src/com/jambit/onboarding2020/tbrpg/domain/Item/EscapeRope.java

```diff
@@ -3,18 +3,20 @@
 import com.jambit.onboarding2020.tbrpg.core.GameState;
 import com.jambit.onboarding2020.tbrpg.domain.Player.Player;
 
-public class EscapeRope extends Item implements Consumable {
 
+public class EscapeRope extends Item implements Consumable {
     Player player = Player.getPlayerInstance();
 
     public EscapeRope() {
-        this.setName("Fluchttrick");
-        this.setLore("Für was dass wohl gut ist?");
+        this.setName("Fluchtseil");
```

Comment:
Wenn wir das mergen, muss im Shop und in GameEngine.interactWithInventory sowie im ItemGenerator noch die Namen von Fluchttrick auf Fluchtseil geändert werden!